### ISSUE TITLE
tracking options "Highlight Selected Item" not working

### ISSUE THREAD

**Issue by SokoFromNZ** [snoopwpf/snoopwpf#391]
Hi,

I've installed the latest alpha yesterday due to [Error snooping .NET 7 app](https://github.com/snoopwpf/snoopwpf/issues/390) but now highlighting seems to be broken.

![Screenshot-20221231-131237](https://user-images.githubusercontent.com/33326277/210137010-d1243a6d-242c-4d66-937c-f04e2fa4805c.png)

The second RadioButton should have the red rectangle around it. Under `Tracking options` `Highlight selected item` is active and the `Highlight options...` are also the default values.

Weirdly enough it seems to work 1 time out of 50 times I'm starting from VS the app and try to snoop it. But I haven't found any logic behind it.

Do you know this issue as well?

PS: Again its latest VS2022 with .NET 7

**Comment by batzen** [comment]
Judging from the screenshot there is no adorner layer available.
There should be a warning shown in the diagnostics pane, about such a missing adorner layer.

**Comment by SokoFromNZ** [comment]
That was the info I needed.
I have had a look at the diagnostics pane once Snoop was attached - no errors. But the error occurs only when I try to highlight a control (which is obvious now).

I have an own `ControlTemplate` for my main window which I've copied from the default template via VS Context-Menu in the XAML designer. Apparently this copy is not the full story of the control :(

Once I've put an `AdornerDecorator` around all my content in the `ControlTemplate` I saw the lovely red rectangle again.

Thanks heaps again!
